webserver/webserver/src/main/java/io/helidon/webserver/ServerResponse.java

```diff
@@ -100,6 +100,27 @@ default ServerResponse status(int statusCode) throws AlreadyCompletedException {
      */
     MessageBodyWriterContext writerContext();
 
+    /**
+     * Send a {@link Throwable} and close the response.
+     *
+     * @param content the {@link Throwable} to send
+     * @return {@code null} when invoked
+     * @throws IllegalArgumentException if there is no registered writer for a given type
+     * @throws IllegalStateException if any {@code send(...)} method was already called
+     * @see #send(Object)
+     */
+    default Void send(Throwable content) {
```

Comment:
Lets not use a default method here. 

The API and implementation come together, so there isn't much benefit to having a default method implementation right ?

Reply:
OK; will change to an interface here and then will move its contents to {checks notes} `Response.java`.